sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRel.java

```diff
@@ -411,6 +414,8 @@ public Expression field(BlockBuilder list, int index, Type storageType) {
                 Expressions.equal(field, Expressions.constant(null)),
                 Expressions.constant(null),
                 Expressions.call(WrappedList.class, "of", field));
+      } else if (fromType.getTypeName() == TypeName.BYTES && storageType == ByteString.class) {
```

Comment:
Had to add "storageType == ByteString.class", "InputGetterImpl" is used to convert Beam schema types to UDF parameters, that we don't always want for our own UDFs.